Cargo.toml

```diff
@@ -37,7 +37,7 @@ winit = "0.30.0" # load icon
 image = "0.25.1" # load icon
 copypasta = "0.10.1" # copy to clipboard
 
-winapi = "0.3.9"
+winapi = { version = "0.3.9", features = ["wincon"] }
```

Comment:
Fixes the failure in build seen here: https://github.com/couleur-tweak-tips/smoothie-rs/actions/runs/9840945429/job/27166377451
```rs
11  |     wincon::GetConsoleWindow,
    |     ^^^^^^ could not find `wincon` in `um`
```